### PR TITLE
Removes trekchems completely

### PR DESCRIPTION
**Pull request by JJRcop**
note from [:grapes:](#JJRcop): *I'm just acting as a relay for [:apple:](#oranges) here and don't really care about this very much. If you want a response from him contact him somewhere he isn't [:banana:](#god).*

## About The Pull Request

Continuing from goof's #44150 all trekchems have been removed and replaced with existing chems. This includes Dexalin and Inaprovaline which didn't have recipes anyway. They have been replaced with ~~Styptic Powder, Silver Sulfadiazine,~~ Salicyclic Acid, Oxandrolone, Salbutamol, Charcoal, and Omnizine.

Closes #44150 

## Why It's Good For The Game

[:tangerine:](https://github.com/tgstation/tgstation/pull/44150#issuecomment-496170534)

## Changelog

:cl: Multiple Contributors
del: Bicaridine, dexalin, kelotane, inaprovaline, and tricordrazine have been removed from the game.
tweak: Usage of these chemicals by game functions have been replaced with existing approximate chemicals.
/:cl:

### CODE REVIEW

File: code/game/machinery/Sleeper.dm [hunk 1]
```diff
@@ -17,12 +17,12 @@
 	var/list/available_chems
 	var/controls_inside = FALSE
 	var/list/possible_chems = list(
-		list("epinephrine", "morphine", "salbutamol", "bicaridine", "kelotane"),
+		list("epinephrine", "morphine", "salbutamol", "styptic_powder", "silver_sulfadiazine"),
```
Comment (Tlaltecuhtli) [inline code/game/machinery/Sleeper.dm]: i dont think you want to inject those

Comment (JJRcop) [reply code/game/machinery/Sleeper.dm]: they do work

File: code/modules/food_and_drinks/food/snacks_other.dm [hunk 2]
```diff
@@ -492,7 +492,7 @@
 	desc = "A colorful, sugary gumball."
 	icon = 'icons/obj/lollipop.dmi'
 	icon_state = "gumball"
-	list_reagents = list("sugar" = 5, "bicaridine" = 2, "kelotane" = 2)	//Kek
+	list_reagents = list("sugar" = 5, "styptic_powder" = 2, "silver_sulfadiazine" = 2)	//Kek
```
Comment (Tlaltecuhtli) [inline code/modules/food_and_drinks/food/snacks_other.dm]: same

File: code/modules/hydroponics/grown/ambrosia.dm [hunk 3]
```diff
@@ -25,7 +25,7 @@
 	icon_dead = "ambrosia-dead"
 	genes = list(/datum/plant_gene/trait/repeated_harvest)
 	mutatelist = list(/obj/item/seeds/ambrosia/deus)
-	reagents_add = list("space_drugs" = 0.15, "bicaridine" = 0.1, "kelotane" = 0.1, "vitamin" = 0.04, "nutriment" = 0.05, "toxin" = 0.1)
+	reagents_add = list("space_drugs" = 0.15, "styptic_powder" = 0.1, "silver_sulfadiazine" = 0.1, "vitamin" = 0.04, "nutriment" = 0.05, "toxin" = 0.1)
```
Comment (Tlaltecuhtli) [inline code/modules/hydroponics/grown/ambrosia.dm]: same

**Comment by HideAndSeekLOGIC** [comment]
I don't see a problem with bicard, kelo and tricord still being in the game.

**Comment by Tlaltecuhtli** [comment]
is this pr ironic or not cause those chems arent for injecting

**Comment by Krysonism** [comment]
The upside of this removal is that it frees up design space.

On the other hand the trekchems were never obsolete and should probably be replaced in the future with more interesting chems.

I have in fact been working on  a more dangerous and unique replacement for anti-toxin for when this goes through.

File: code/game/machinery/Sleeper.dm (hunk 1 above)
Comment (Ghommie) [reply code/game/machinery/Sleeper.dm]: Styptic powder and silver sulfadiazine currently cause an instant tox loss equal to half the volume on reaction_mob() if the method is VAPOR, INJECT or INGEST.
So yea, good luck unless you remove its tox loss quirk.

**Comment by Angustmeta** [comment]
There are no close counterparts for some of them, but the closest available chems would be:
- Bicaridine -> Salicyclic Acid
- Kelotane -> Oxandrolone

I'm curious about how Goon heals most of their brute and burn. Is cryo meta there? They seem to have tricordrazine, except there it's called Saline-Glucose Solution (much stronger version than ours).

File: code/game/machinery/Sleeper.dm (hunk 1 above)
Comment (81Denton) [reply code/game/machinery/Sleeper.dm]: This really makes me think

**Review by 81Denton — CHANGES_REQUESTED** [review]
Please either 
1) Replace styptic/silver sulf with equivalents that don't kill people when injected OR
2) Rework both chems to not kill people when injected

**Comment by Barhandar** [comment]
@81Denton hugbox; what's a little death from medibots spewing toxins for a good spessman?

**Comment by JJRcop** [comment]
Thanks @Angustmeta changes coming shortly

**Comment by JJRcop** [comment]
Complied to @81Denton's request using @Angustmeta's suggestion. Styptic powder use replaced with Salicyclic Acid, Silver Sulfadiazine use replaced with Oxandrolone.

**Comment by 0d0be32b607ee4ecbc2358f3dd4a27de** [comment]
WEE WOO WEE WOO INB4 ORANGE BUS

**Comment by Barhandar** [comment]
Boy oh boy medbots that can heal 4 damage per 2 seconds

**Comment by Shadowflame909** [comment]
JJRcop BRUTALLY murders the competition with another nerf PR.

**Comment by Akrilla** [comment]
Can we stop blueballing each other and just delete medbay?

**Comment by JJRcop** [comment]
@optimumtact Do you have any interest in this?

**Comment by OverlyGeneric** [comment]
>mfw when reddit is complaining about sleepers

>mfw when we're already removing trekchems and are planning to kill off cloning.

>mfw when we're going down a dark path of crippling medbay's capabilities.

image

**Comment by Shadowflame909** [comment]
Mfw all antags still have the potential to murder 1/3rd of the crew at the very least but medbay is getting beaten so badly

**Comment by anconfuzedrock** [comment]
me realising easier access to pent acid and oxy is really a medbay buff
image

**Comment by optimumtact** [comment]
stop making pr's you don't care about, I swear to god

**Pull request closed without merging**